python/_restclient/setup.py

```diff
@@ -13,17 +13,21 @@
 SUBPACKAGE_SLUG = f"{NAMESPACE_PACKAGE_NAME}.{SUBPACKAGE_NAME}"
 
 # Subpackage version
-VERSION = "2.1.0-alpha.0"
+VERSION = "3.0.0"
 
 # Package author information
 AUTHOR = "Austin Raney"
 AUTHOR_EMAIL = "arthur.raney@noaa.gov"
 
 # Short sub-package description
-DESCRIPTION = "Abstract REST api client with built in request caching and retries"
+DESCRIPTION = (
+    "Utilities adept for quickly and succinctly writing REST api client libraries."
```

Comment:
Honestly, I think the old description was more helpful.

Reply:
Okay, I may just drop "Abstract" as it can have undesired connotations. But I am happy to roll it back.

Reply:
I wasn't really advocating for a "roll back." What's nice about the old description is that it's more explicit about the capabilities offered by the package while also being succinct.